README.md

```diff
@@ -1 +1,17 @@
-# codacy-test
\ No newline at end of file
+# codacy-test
+
+### Among Codacy’s features
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://release.k8s.dev.codacy.org/gh/DevK8scodacytestuser/codacy-test/pullRequest?prid=3215)
```suggestion
### Among Codacy’s features

```